jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+let invertedIndexTest = new InvertedIndexClass();
```

Comment:
You should preferably use `const` instead of `let` here seeing as the variable `invertedIndexTest` is not reassigned.